play/PlayerRules/Basic Rules.txt

```diff
@@ -581,13 +604,14 @@ to roll either for (15 - FORT) turns, or until they pass.
 Conditions:
 Bleeding - d10/2+1 (drop the remainder) stackable damage per turn. Broken with 
 Medicine Skill check. DC varies. If you have a bleed condition, seek medical aid
-immediately, You CAN Bleed Out, you have been warned.
+immediately, You can Bleed Out.
 Burning - 2d10 damage per turn. Broken with a Maneuver Skill Check. DC 40. 
 Every 2 turns, the degree of the burn intensifies
 Downed - Your health has fallen below 0. Every turn in this condition, you 
 must pass a DC 70 Fort save. You are allowed a number of failures equal to 
-your Fort score. when you have failed more times than your Fort score, you 
-die. While in this state, you cannot take any actions, but can mumble or speak 
+your Fort score, for your lifespan. when you have failed more times than 
+your Fort score, you die. While in this state, you cannot take any actions,
+but can mumble or speak 

```

Comment:
although i made the boo-boo of thinking that we weren't playing according to the rules with #133